ui/app/peers/[peerName]/lagGraph.tsx

```diff
@@ -81,20 +78,26 @@ function LagGraph({ slotNames }: { slotNames: string[] }) {
         <ReactSelect
           className='w-1/4'
           placeholder='Select a replication slot'
-          options={slotNames.map((slotName) => ({
-            label: slotName,
-            value: slotName,
-          }))}
+          options={
+            slotNames.map((slotName) => ({
+              label: slotName,
+              value: slotName,
+            })) ?? undefined
```

Comment:
```suggestion
            }))
```
`Array.prototype.map` will always return an array

Reply:
What I want is if the array is empty then return undefined and not empty array. This is because react-select placeholder doesn't appear otherwise

Reply:
You'll have to do `slotNames.length === 0 ? undefined : slotNames.map(...)`